docker/development/compose.yml

```diff
@@ -45,7 +45,7 @@ services:
     volumes:
       - "../../backend/:/app/"
       - "./backend/scripts/:/scripts/"
-      - "etc-template-backend-cache:$HOME/.cache/pip"
+      - "etc-template-backend-cache:/c/Users/taral/.cache/pip"
```

Comment:
@taralalukasz 
Please don't commit that change.
You can keep that localy, but please remember that 80% developers are using Linux.